### ISSUE TITLE
Support Multiplet Lineshapes for divertor spectroscopy

### ISSUE THREAD

**Issue by mattngc** [cherab/core#69]
@shenders has expressed interest in having support for multiplet line shapes in CHERAB. Although it would be possible to add this functionality to atomic data providers, such as OpenADAS, it seems this data isn't widely available and rather specialist.

Instead, we propose to make a new Multiplet() lineshape object. It could take as argument a list of wavelengths with the associated multiplet ratios.

This issue will track all requirements for the functionality.

**Comment by mattngc** [comment]
@shenders, I've added a draft implementation in the feature/multiplet branch. You can test it with code like this:
```
       >>> from cherab.core.atomic import Line, deuterium
       >>> from cherab.core.model import ExcitationLine, MultipletLineShape
       >>>
       >>> # multiplet specification in Nx2 array
       >>> multiplet = [[403.5, 404.1, 404.3], [0.2, 0.5, 0.3]]
       >>>
       >>> # Adding the multiplet to the plasma model.
       >>> d_alpha = Line(deuterium, 0, (3, 2))
       >>> excit = ExcitationLine(d_alpha, lineshape=MultipletLineShape, lineshape_args=[multiplet])
       >>> plasma.models.add(excit)
```

**Comment by mattngc** [comment]
Resolved in PR #70.
